### DISCUSSION TITLE
Dbeast monitor v0.7.0 Released!

### DISCUSSION THREAD

**Discussion by dbeast-co** [dbeast-co/dbeast-monitor#50] [Announcements]
Dbeast monitor v0.7.0 Released!
Full Changelog: https://github.com/dbeast-co/dbeast-monitor/wiki/ReleaseNotes
Installation Instructions: https://github.com/dbeast-co/dbeast-monitor/wiki/Installation-Instructions

**Comment by latsygin** [comment]
Is this tool applicable to Opensearch clusters?

**Reply by dbeast-co** [reply]
Not yet, but we have this in the roadmap.
